lib/cli/run.js

```diff
@@ -0,0 +1,403 @@
+        group: GROUPS.RULES,
+        type: 'boolean'
+      },
+      'expose-gc': {
+        alias: 'gc',
+        description: 'Equivalent to node flag "--expose-gc"',
+        hidden: true,
+        type: 'boolean'
+      },
+      'forbid-only': {
+        description: 'Fail if exclusive test(s) encountered',
+        group: GROUPS.RULES,
+        type: 'boolean'
+      },
+      'forbid-pending': {
+        description: 'Fail if pending test(s) encountered',
+        group: GROUPS.RULES,
+        type: 'boolean'
+      },
+      'full-trace': {
+        description: 'Display full stack traces',
+        group: GROUPS.OUTPUT,
+        type: 'boolean'
+      },
+      'inline-diffs': {
+        description:
+          'Display actual/expected differences inline within each string',
+        group: GROUPS.OUTPUT,
+        type: 'boolean'
+      },
+      'no-colors': {
+        alias: 'C',
+        conflicts: ['colors'],
+        description: 'Disable color output',
+        group: GROUPS.OUTPUT,
+        type: 'boolean'
+      },
+      'no-timeouts': {
+        conflicts: ['timeout'],
+        description: 'Disable timeouts',
+        group: GROUPS.RULES,
+        type: 'boolean'
+      },
+      'reporter-options': {
+        alias: 'O',
+        coerce: (opts = '') =>
+          opts.split(',').reduce((acc, opt) => {
+            const pair = opt.split('=');
+
+            if (pair.length > 2 || !pair.length) {
+              throw new Error(`invalid reporter option '${opt}'`);
+            }
+
+            acc[pair[0]] = pair.length === 2 ? pair[1] : true;
+            return acc;
+          }, {}),
+        description: 'Reporter-specific options (<k=v,[k1=v1,..]>)',
+        group: GROUPS.OUTPUT,
+        requiresArg: true,
+        type: 'string'
+      },
+      'watch-extensions': {
+        coerce: list,
+        default: 'js',
+        description:
+          'Comma-separated list of extensions to monitor with "--watch"',
+        group: GROUPS.FILES,
+        requiresArg: true,
+        type: 'string'
+      },
+      bail: {
+        alias: 'b',
+        description: 'Abort ("bail") after first test failure',
+        group: GROUPS.RULES,
+        type: 'boolean'
+      },
+      colors: {
+        alias: 'c',
+        conflicts: ['no-colors'],
+        description: 'Force-enable color output',
+        group: GROUPS.OUTPUT,
+        type: 'boolean'
+      },
+      compilers: {
+        coerce: (opts = '') => opts.split(','),
+        description:
+          'Use the given module(s) to compile files (<ext:module,[ext1:module1]>)',
+        group: GROUPS.FILES,
+        requiresArg: true,
+        type: 'string'
+      },
+      config: {
+        config: true,
+        default: CONFIG_PATH_PLACEHOLDER,
+        defaultDescription: '(nearest rc file)',
+        description: 'Load config from file',
+        global: false,
+        group: GROUPS.CONFIG
+      },
+      debug: {
+        alias: 'd',
```

Comment:
Can we drop the `d` alias?

Reply:
I realize also that `debug` (not `--debug`, but does same thing) is a valid command which I failed to implement.

- [ ] drop `-d` alias for `--debug`
- [ ] note that `debug` is no longer supported somewhere